screed/tests/test_attriberror.py

```diff
@@ -0,0 +1,85 @@
+import screed
+from screed.DBConstants import fileExtension
+import os
+
+
+class nostring:
+    def __init__(self):
+        self.exists = True
+
+    def __str__(self):
+        raise AttributeError
```

Comment:
Stepping back from the tests we need to determine if the expected behavior is correct or not; my opinion is that the expected behavior is wrong. This is the value of writing tests because it helps you re-evaluate code from another perspective.